extractor/src/extractors/JsonnetfileJsonExtractor.ts

```diff
@@ -0,0 +1,41 @@
+import MatchConfig from "../matcher/MatchConfig";
+import Extractor from "./Extractor";
+import ExtractorFile from "./ExtractorFile";
+import Languages from "./Languages";
+
+const extract = (dependency: any) => {
+    const { subdir , name } = dependency.source.git
+    return {
+        subdir,
+        name,
+    };
+}
```

Comment:
This should return the standard Dependency structure like the others. (see the type annotations)

```
const extract = (dependency: any): Dependency => {
  const { source, version } = dependency;
  const { git } = (source || {});
  const { subdir, name } = (git || {});

  return {
    name,
    versionConstraint: version,
    scopes: subdir ? [ subdir ] : [],
    organization: "", // deprecated
    module: "", // deprecated
  };
};
```